Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -703,51 +703,118 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
 
   /// ``CaseConversionStrategy`` is used to specify the strategy used to convert the casing of
   /// GraphQL schema values into generated Swift code.
-  public enum CaseConversionStrategy: String, Codable, Equatable {
-    /// Generates swift code using the exact name provided in the GraphQL schema
-    /// performing no conversion.
-    case none
-    /// Convert to lower camel case from `snake_case`, `UpperCamelCase`, or `UPPERCASE`.
-    case camelCase
+  @available(*, deprecated, message: "Use EnumCaseConversionStrategy instead.")
+    public enum CaseConversionStrategy: String, Codable, Equatable {
```

Comment:
We should move this to the bottom of the file where the other deprecated stuff lives.